YogaKit/Source/YGLayout.m

```diff
@@ -304,10 +304,15 @@ static YGSize YGMeasureView(
   const CGFloat constrainedHeight = (heightMode == YGMeasureModeUndefined) ? CGFLOAT_MAX: height;
 
   UIView *view = (__bridge UIView*) YGNodeGetContext(node);
-  const CGSize sizeThatFits = [view sizeThatFits:(CGSize) {
-    .width = constrainedWidth,
-    .height = constrainedHeight,
-  }];
+  CGSize sizeThatFits = CGSizeZero;
+
+  // Fix for https://github.com/facebook/yoga/issues/606
+  if (![view isMemberOfClass:[UIView class]] || [view.subviews count] > 0) {
```

Comment:
I wonder if there is a cleaner way to do this...I don't want us to check the class of a view every time yoga measures it.